source/Octopus.Client/IOctopusAsyncRepository.cs

```diff
@@ -7,59 +9,11 @@ namespace Octopus.Client
     /// Functionality not exposed by this interface can be accessed
     /// using <see cref="IOctopusAsyncRepository.Client" />.
     /// </summary>
-    public interface IOctopusAsyncRepository
+    public interface IOctopusAsyncRepository : IAsyncSpaceRepository, IAsyncSystemRepository
```

Comment:
Very minor thing - `IAsyncSpaceRepository` seems inconsistently named from `IOctopusAsyncRepository` - wondering if `ISpaceAsyncRepository` is more consistent?